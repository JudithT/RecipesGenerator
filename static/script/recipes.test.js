describe("It search a recipe by name or by ingredients", () => {
    test('thing', () => {
        expect(2).toMatchSnapshot()
    })
})