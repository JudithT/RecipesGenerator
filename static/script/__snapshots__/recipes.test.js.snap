// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It search a recipe by name or by ingredients thing 1`] = `1`;
